modules/jaggery-apps/app-store-web/themes/store/js/assets.js

```diff
@@ -133,3 +133,28 @@ $(function() {
 	caramel.loaded('js', 'assets');
 	caramel.loaded('js', 'sort-assets');
 });
+
+var mouseStop = function() {
```

Comment:
Method name is confusing. It binds the mouseStop and mouseLeave events. But the method names doesn't reflect it.